spec/models/category_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+describe Category do
+  it { should have_many(:videos)}
+
+  describe "recent_videos" do
```

Comment:
The issue is that I don't know what method is that - class or instance? `describe "search_by_title"` Its easier to annotate your methods and driving RSpecs style on describing. Prepending `.` would be a class method and `#` would be an instance method. Read in the link I mentioned.